contracts/unstoppable/FirstChallenge.conf

```diff
@@ -0,0 +1,15 @@
+{
+  "files": [
+    "contracts/unstoppable/ReceiverUnstoppable.sol",
+    "contracts/unstoppable/UnstoppableLender.sol",
+    "contracts/DamnValuableToken.sol"
+  ],
+  "verify": "ReceiverUnstoppable:contracts/unstoppable/FirstChallenge.spec",
+  "rule_sanity": "basic",
+  "link": [
+        "ReceiverUnstoppable:pool=UnstoppableLender",
+        "UnstoppableLender:damnValuableToken=DamnValuableToken"
+    ],
+  "optimistic_loop": true,
+  "parametric_contracts" : ["ReceiverUnstoppable", "DamnValuableToken"]
```

Comment:
ReceiverUnstoppable is filtered out? it's the main contract, no?

Reply:
I removed the parametric contract it's not needed.